cells/lib/ops/mkStandardOCI.nix

```diff
@@ -68,19 +68,43 @@ in
       then debugOperable
       else operable;
 
-    setupLinks = cell.ops.mkSetup "links" [] ''
-      mkdir -p $out/bin
-      ${runtimeEntryLink}
-      ${debugEntryLink}
-      ${livenessLink}
-      ${readinessLink}
-    '';
+    setupLinks =
+      cell.ops.mkSetup "links" [
+        {
+          regex = "/bin";
+          mode = "0555";
```

Comment:
here this means `rx-rx-rx` on `/bin`.

A suitable restriction?

Is there an escape hatch? Yes: `[{regex = "/bin/mything"; mode = "0100";}] ":"`